fe/fe-core/src/main/java/org/apache/doris/clone/ColocateTableCheckerAndBalancer.java

```diff
@@ -437,25 +454,35 @@ private Set<GroupId> relocateAndBalanceGroup(Set<GroupId> groupIds, boolean bala
                 Set<Long> beIdsInOtherTag = colocateIndex.getBackendIdsExceptForTag(groupId, tag);
                 List<Long> availableBeIds = getAvailableBeIds(SystemInfoService.DEFAULT_CLUSTER, tag, beIdsInOtherTag,
                         infoService);
+
+                Set<Long> nonAliveBeIds = getNonAliveBeIds(unavailableBeIdsInGroup, infoService);
+                if ((CollectionUtils.isEmpty(unavailableBeIdsInGroup) || nonAliveBeIds.isEmpty())
```

Comment:
if a be is decommission, should move its tablets to other BE as soon as possible.